src/generators/operationGenerator.ts

```diff
@@ -894,22 +868,15 @@ function getSpanStatusCode() {
 function getTracingTryCatchStatement(
```

Comment:
Since we don't longer use a try catch statement, should this function be renamed?

Reply:
Maybe something like `getTracingClientWithSpanStatement`?